app/models/order.rb

```diff
@@ -7,7 +7,7 @@ class Order < ActiveRecord::Base
   validates :total_cost, :user_id, presence: true
 
   def self.complete
-    all.select { |order| order.status == 'complete'}
+    all.select { |order| order.status == 'completed'}
```

Comment:
Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.